.github/TEST_CONTRIBUTING/TEST_GUIDELINE.md

```diff
@@ -0,0 +1,145 @@
+
+# Contributing Guideline - Writing Tests for Design System Components
+
+## Vue3 New Test Template
+[test_template](test_template.js)
+
+This document aims to provide comprehensive information for effectively creating tests that ensure the quality, reliability, and consistency of our components.
+
+To help you get started quickly, we have prepared a standardized test template.
+
+You can also check these tests to get familiar with how it is working now in [checkbox-component-test](../../components/checkbox/checkbox.test.js) or [icon-component-test](../../components/icon/icon.test.js)
+
+This template provides an initial scaffolding and follows the structure we expect for our tests.
+
+This template is also generated by yeoman which creates a new component with `npx yo dialtone-vue` script.
+
+
+## Organizing Variables for Effective Testing:
+[structuring_variables](assets/structuring_variables.png)
```

Comment:
Add an ! (exclamation mark) at the beginning of the images to inline them instead of making them links
```suggestion
![structuring_variables](assets/structuring_variables.png)
```

Reply:
I did it in the beginning but I didn't like how its shows due different sizes. 
I would prefer to keep it as it.

Reply:
You can use HTML in markdown to make it the appropriate size if needed.

It's not usable that way, If you are going to include images, I'd like to see the images there, not links that take me out of the current docs.

https://stackoverflow.com/a/14747656/16506300

Reply:
I tryed with html also... width of 200, still does not convince due different sizes of images.
Also you cannot zoom it.

As I tested previously when you open the reference, it will open you in a different window I didn't see anything wrong besides normal behaviour.

I still see it more properly with links instead of images.

But as I see it is a required demand, let me adjust them individually to look it ok.

Reply:
Since it impact the readability of the document I will ping @braddialpad here to check it out.

Reply:
It looks much better when the images are embedded. However why are we even using screenshots for code examples? Shouldn't we use a markdown code block? This is much more maintainable.

Reply:
lol right, lets move it to code block

Reply:
Ah right, I haven't thought about code blocks instead of images. That'd be better, IDK if it's possible to import the lines of code from the file, that'd be even better.

Reply:
Updated using codeblocks

Reply:
good just need to update code language as mentioned below